domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraEvent.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.jetpackcamera.domain.camera
+
+import androidx.camera.core.MeteringPoint
+
+sealed interface CameraEvent
+
+class FocusMeteringEvent(val meteringPoint: MeteringPoint) : CameraEvent
```

Comment:
nit: We've been putting our sealed interface implementations nested in the sealed interface itself. For consistency, can we change this to:
```
sealed interface CameraEvent {
  class FocusMetering(...) : CameraEvent
}
```